packages/api/src/hooks/useMT5AccountsList.ts

```diff
@@ -52,7 +52,7 @@ const useMT5AccountsList = () => {
 
     return {
         /** The list of created MT5 accounts */
-        data: modified_mt5_accounts,
+        data: modified_mt5_accounts ?? [],
```

Comment:
```suggestion
        data: modified_mt5_accounts,
```
Do not change this! By design, consuming developer should check for null

Reply:
We can revert this file change as we are not using this hook in this PR